bin/cron/delete_twilio_data

```diff
@@ -6,15 +6,18 @@
 
 # NOTE: As ListResource.list(), by default, returns the most recent fifty
 # instances, we wrap list() methods in while loops to assure all appropriate
-# records are processed.
+# records are processed. This is unnecessary for media, as a message may contain
```

Comment:
Note that we can specify a page size. Is there benefit or cost to specifying the maximum allowed page size of `1000` when calling `messages.list`?